app/models/concerns/admin/businesses/hiring_invoice_requests/notifications.rb

```diff
@@ -0,0 +1,18 @@
+module Admin
```

Comment:
I don't think this should be in the Admin namespace. Yes, we are sending a notification to an admin but that's more implementation details in my opinion.

Reply:
changed in e648e383ab02f21a0dbe2e46495bbf7df2df2a3b